.husky/pre-push

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+validate_branch_name() {
+  local branch=$(git rev-parse --abbrev-ref HEAD)
+  local regex="^main|((feat|fix|refactor|perf|tech|docs|renovate|ci|security|style|test|chore)\/([0-9]+)|(feat|fix|refactor|perf|tech|docs|renovate|ci|security|style|test|chore)\/([0-9]+\/)?([a-z0-9-]+)|(release|hotfix)\/([0-9.]+))$"
```

Comment:
@ResistantBear @mylinhdao Please align the allowed prefixes here with the contribution guide.

fyi: be aware that this allows branch names like `feat/12345` without any description

Reply:
I have updated that regex and the documentation below it to match our naming conventions.